### ISSUE TITLE
Talkback does not handle HTML implicit labels correctly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [2php/eyes-free#490]
```
1. Go to a page with this kind of HTML:

  <div>
    <label>
      <span>Implicit</span>
      <input name="in1" id="TextBox_t1" type="text" maxlength="17">
    </label>
  </div>

e.g. https://output.jsbin.com/fujahi

2. Use TalkBack to navigate to the input field


What is the expected output? What do you see instead?

Talkback should announce the element and the label "Edit Box. Implicit"

Instead, TalkBack just announces "Edit Box"

If an explicit label is used, TalkBack works correctly:

  <div>
    <label for="TextBox_t2">
      <span>Explicit</span>
    </label>

    <input name="in2" id="TextBox_t2" type="text" maxlength="17">
  </div>

If an implicit label has explicit semantics added, TalkBack ignores the 
presence of the explicit labeling:

  <div>
    <label for="TextBox_t3">
      <span>Implicit and Explicit</span>
      <input name="in3" id="TextBox_t3" type="text" maxlength="17">
    </label>
  </div>


What version of the product are you using? On what operating system?

Android 6.0 with TalkBack 4.3.1 and Chrome 46.0.2490.76 on a Nexus 5.

```

Original issue reported on code.google.com by `Owen.R.E...@gmail.com` on 25 Nov 2015 at 11:46

**Comment by GoogleCodeExporter** [comment]
```
Most likely, this is a Chrome bug, which should be filed at crbug.com. System 
Web View may not be supplying necessary info. 
```

Original comment by `KevinCha...@gmail.com` on 23 Jan 2016 at 8:43
